### ISSUE TITLE
Hotspot menu - minor issue with sighted user parity

### ISSUE THREAD

**Issue by oliverfoster** [cgkineo/adapt-hotspotMenu#1]
_From @shauncg on October 28, 2014 16:18_

Overall the hotspot menu works well with JAWS. However, when a hotspot is selected and the pop-up appears containing additional information on the topic, JAWS will only read the 'View' button to which the cursor is by default set. 

In terms of functionality this is fine and should be clear for a JAWS user. However, they will not know that there is any additional information available to them e.g. by tapping the up key to step back through the newly displayed content. 

(To be honest I think this is okay. I'm only reporting it because it potentially excludes some content from visually impaired users. However it's arguably non-essential content and I can't think of a more elegant way of having this content read to JAWS users and still retaining the functionality of having the View button selected and ready to go.)

_Copied from original issue: cgkineo/adapt_framework#35_
